wiki/en/en.FAQ.md

```diff
@@ -0,0 +1,48 @@
+
+### How do I know if I can join a server? Are there rules?
+
+In general, if somebody sets up a public server, then [by definition](/wiki/Choosing-a-Server-Type) they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. However, some servers may state their policies in the welcome message you will see in the chat window.
+
+You can also set up a private server on Jamulus and give others your address to connect to that. Have a look at [this page for more information](/wiki/Running-a-Server).
+
+### Why shouldn’t I use wireless equipment?
+
+Whether or not you will be able to play in time with other musicians depends mainly on how much latency you have in your sound signal. An overall delay much bigger than 50ms usually makes it too hard to play in time unless you train yourself to do so. Some musicians report they can play in time with delays of up to 70ms by playing ahead of the beat. But much beyond that and it becomes too difficult unless timing isn’t an important factor in the music.
+
+So it makes sense to minimise any sources of delay or other problems with the signal. Even fast wi-fi is usually too variable to use for long periods with Jamulus, and Bluetooth headphones and mics usually introduce latencies of about 50ms or more. 
+
+### Why should I not listen to my own signal?
+
+For the same reason as you need to minimise delay in your signal in order to play in time, you need to make sure you are playing to your own sound that other musicians are hearing. More information on this, and a way of testing your setup to make sure you are obeying this “Golden Rule” [can be found here](/wiki/Client-Troubleshooting).
+
+Of course, if you are playing an acoustic instrument, or are a singer, it will be hard to exclude your “local” sound. But you should at least try to do so by for example using closed back headphones turned up as loud as you can to mask your own sound so that you can concentrate on the mix coming back to you from the server.
+
+### Do I need a fast Internet connection?
+
+For most people on standard broadband (e.g. 10 Mbit/s down and 1 Mbit/s up) you will have no problems. For those running servers at home, depending on how many people join, you may encounter issues if your upstream bandwidth is lower than about 5 Mbit/s. [More information on network requirements here](/wiki/Network-Requirements).
+
+### Do I need to run a server?
+
+No. If you just intend to connect to other people’s servers, then all you need is a client. [Read this if you think you need to run your own server](/wiki/Running-a-Server). 
+
+### How does Jamulus work (in general)?
+
+[diagram]
+
+Jamulus works on the client-server principle. Everybody’s audio is sent to a server, mixed and processed there. Afterwards, the audio is sent back to each client. If a server is made public and registered on a central server, its information will be broadcast to all clients.
+
```

Comment:
... does Jamulus provide Video support? 

No. You can use other software like Jitsi or Zoom

... I have trouble with my audio: I hear a loud squeal or nothing at all:

Your audio devices aren't correctly set up... see the troubleshooting page,

Reply:
I'll add the video stuff, but "having trouble with X" is for Troubleshooting. This FAQ is just a page for general  "I'm curious/want to know more" stuff.

Reply:
Ok. Yes. However I think that's one of the main issues people face on their first install?

Reply:
They won't be following a link that says something like "If you have problems using Jamulus". If they are that's a bug :-) The FAQ is a place where we can put all the bits and pieces that people ask about that aren't about using Jamulus itself. For usage problems they should instead be directed to Troubleshooting.  I'll add a link to that though at the end.